instrumentation/graphql-java/graphql-java-20.0/library/src/main/java/io/opentelemetry/instrumentation/graphql/v20_0/OpenTelemetryInstrumentation.java

```diff
@@ -8,22 +8,65 @@
 import static graphql.execution.instrumentation.InstrumentationState.ofState;
 
 import graphql.ExecutionResult;
+import graphql.GraphQLError;
+import graphql.execution.ResultPath;
 import graphql.execution.instrumentation.InstrumentationContext;
 import graphql.execution.instrumentation.InstrumentationState;
+import graphql.execution.instrumentation.SimpleInstrumentationContext;
 import graphql.execution.instrumentation.SimplePerformantInstrumentation;
 import graphql.execution.instrumentation.parameters.InstrumentationCreateStateParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationExecuteOperationParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationExecutionParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters;
+import graphql.language.AstPrinter;
+import graphql.language.AstTransformer;
+import graphql.language.BooleanValue;
+import graphql.language.EnumValue;
+import graphql.language.Node;
+import graphql.language.NodeVisitor;
+import graphql.language.NodeVisitorStub;
+import graphql.language.NullValue;
+import graphql.language.OperationDefinition;
+import graphql.language.Value;
+import graphql.language.VariableReference;
 import graphql.schema.DataFetcher;
-import io.opentelemetry.instrumentation.graphql.internal.OpenTelemetryInstrumentationHelper;
-import io.opentelemetry.instrumentation.graphql.internal.OpenTelemetryInstrumentationState;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.util.TraversalControl;
+import graphql.util.TraverserContext;
+import graphql.util.TreeTransformerUtil;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.semconv.ExceptionAttributes;
+import java.util.Locale;
+import java.util.concurrent.CompletionStage;
 
 final class OpenTelemetryInstrumentation extends SimplePerformantInstrumentation {
-  private final OpenTelemetryInstrumentationHelper helper;
 
-  OpenTelemetryInstrumentation(OpenTelemetryInstrumentationHelper helper) {
-    this.helper = helper;
+  private static final NodeVisitor sanitizingVisitor = new SanitizingVisitor();
```

Comment:
You may have noticed that in this PR in `build.gradle.kts` I've removed the dependency on `graphql-java-common:library` which prevents me from using methods in `OpenTelemetryInstrumentationHelper`.

I did this because there were changes that I needed to make to `OpenTelemetryInstrumentationState` that did not work with the older version of the GraphQL library which `OpenTelemetryInstrumentationHelper` depends on.

To make the code easier to reason and work with, I decided to copy over a bunch of code from `graphql-java-common:library`. It is possible to make `OpenTelemetryInstrumentationState` more generic across different versions of GraphQL but I found that added more complexity. I would expect this complexity to potentially increase if we needed to adapt the instrumentation to newer versions of GraphQL.